### ISSUE TITLE
Missing 'git flow release abandon' command

### ISSUE THREAD

**Issue by petervanderdoes** [petervanderdoes/gitflow#186]
Quote from original report by @kretes at nvie/gitflow#358

> When using branching model it may happen that release is created, some commits are applied to it, but then it is abandoned - i.e. not deployed to production, but all the fixes should be back-merged to develop. I don't see a way to do it with git flow, so I fallback to doing it by hand

**Comment by jkankiewicz** [comment]
Isn't this functionality already provided by the `git flow release delete <name>` command and the shortcut `git flow delete` command (when you're already on the release branch that you wish to abandon)?

**Comment by wilbertjoosen** [comment]
Hi,

Is this funcionality already provided? I can cancel  RC’s?

**Comment by jkankiewicz** [comment]
> ...all the fixes should be back-merged to develop.

@wilbertjoosen, You're right; a new command is needed because the `delete` command won't merge any commits that are unique to the branch into its parent branch before it is deleted.
Either that or a `--preserve-commits` flag (or something like it) will have to be added to the `delete` command.
The tricky thing is that you don't want it to preserve any commits that update the version number because those should only ever originate from a release branch.
It seems to me that if this command (or flag) is added then the merging process would have to be interactive so that the user will have an opportunity to skip over any commits that update the version number prematurely.
